scripts/env-check.js

```diff
@@ -0,0 +1,21 @@
+const availableVars = [
+  "E2E_PROJECT_ID",
+  "NODE_TLS_REJECT_UNAUTHORIZED",
+  "JEXIA_DEV_DOMAIN",
+  "E2E_EMAIL",
+  "E2E_PASSWORD",
+  "RECAPTCHA_TOKEN",
```

Comment:
for JFS tests we also need a bunch of `AWS_...` variables:
`AWS_KEY, AWS_SECRET, AWS_BUCKET`